QUALITY_DECLARATION.md

```diff
@@ -80,11 +80,28 @@ There is an automated test which runs a linter that ensures each file has at lea
 
 `libyaml_vendor` is a package providing solely CMake files and therefore does not require API or feature tests and has no coverage or performance requirements (sections [4.i - 4.iv]).
 
+### Feature Testing [4.i]
+
+As we mentioned in the Quality Declaration of `libyaml`, ROS 2 only uses the API for reading YAML files. This will be the only `libyaml` API tested in the ROS 2 buildfarm.
+
+### Public API Testing [4.ii]
+
+As we mentioned in the Quality Declaration of `libyaml`, ROS 2 only uses the API for reading YAML files. This will be the only `libyaml` API tested in the ROS 2 buildfarm.
+
+### Coverage [4.iii]
+
+Lastest code coverage can be found [here](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/).
+
+### Performance [4.iv]
+
+The `libyaml` package is benchmarked and the most recent test results can be found [here](http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/BenchmarkTable/).
```

Comment:
I searched for `libyaml` in that report and couldn't find anything. Is there a way to check that the performance tests are running before merging this PR?

Reply:
this PR includes the benchmark test, I can split it in two